Readme.md

```diff
@@ -52,15 +52,13 @@ ready for production use.
 
 ### Moya version vs Swift version.
 
-Because of the many Swift versions Moya supports, it might be confusing to
-find the version of Moya that you need. Below is a table that shows which version of Moya
-you should use for your Swift version.
+Below is a table that shows which version of Moya you should use for
+your Swift version.
 
 | Swift version | Moya version  |
 | ------------- | ------------- |
-| 3.X           | >= 8.0.0      |
-| 2.3           | 7.0.2 - 7.0.4 |
-| 2.2           | <= 7.0.1      |
+| 4.X           | >= 9.0        |
```

Comment:
4.X would be supported probably only for Moya/Core, not for Moya/RxSwift (dunno how about Moya/ReactiveSwift, but probably the same situation as with RxSwift). Not sure how to describe it in this table, though.

Reply:
That would be a shame for RxSwift. I suggest you reconsider supporting Swift 4.

Reply:
Because of waiting for the dependencies to update to Swift 4? Or why?

Reply:
@BasThomas @Coeur This is because we can't support two major versions of reactive extensions libraries at once. We could support Swift 4 in our core with very little change in our codebase, so we did it. Of course we will create a new version with new major versions of reactive extensions libraries just after 9.0.0, where beta might be really soon available.

About the table, maybe we should split the Moya version column on Moya and Moya/RxSwift, Moya/ReactiveSwift.